Packs/AnsibleTower/Integrations/AnsibleTower/AnsibleTower.py

```diff
@@ -0,0 +1,466 @@
+                    empty_valid_codes: list = None, return_empty_response: bool = False, ok_codes: list = None) -> dict:
+        return self._http_request(method=method, url_suffix=url_suffix, params=params, json_data=json_data,
+                                  empty_valid_codes=empty_valid_codes, return_empty_response=return_empty_response,
+                                  ok_codes=ok_codes)
+
+
+def inventories_list(client: Client, args: dict) -> List[CommandResults]:
+    command_results = []
+    response = client.api_request(method='GET', url_suffix='inventories/', params=args)
+    results = response.get('results', [])
+    if not len(results):
+        command_results.append(CommandResults(
+            readable_output=f"No results were found for the following arguments {str(args)}",
+            raw_response=response
+        ))
+    for res in results:
+        res.pop('related', None)  # remove irrelevant fields from output
+        res.pop('summary_fields', None)
+        command_results.append(CommandResults(
+            outputs_prefix='AnsibleAWX.Inventory',
+            outputs_key_field='id',
+            outputs=res,
+            readable_output=tableToMarkdown(name='Results', t=res, removeNull=True),
+            raw_response=res
+        ))
+    return command_results
+
+
+def hosts_list(client: Client, args: dict) -> List[CommandResults]:
+    inventory_id = args.pop('inventory_id', None)
+    if inventory_id:
+        url_suffix = f'inventories/{inventory_id}/hosts/'
+    else:
+        url_suffix = 'hosts/'
+
+    command_results = []
+    response = client.api_request(method='GET', url_suffix=url_suffix, params=args)
+    results = response.get('results', [])
+    if not len(results):
+        command_results.append(CommandResults(
+            readable_output=f"No results were found for the following argument {str(args)}",
+            raw_response=response
+        ))
+    for res in results:
+        res.pop('related', None)  # remove irrelevant fields from output
+        res.pop('summary_fields', None)
+        command_results.append(CommandResults(
+            outputs_prefix='AnsibleAWX.Host',
+            outputs_key_field='id',
+            outputs=res,
+            readable_output=tableToMarkdown(name='Results', t=res, removeNull=True),
+            raw_response=response
+        ))
+    return command_results
+
+
+def create_host(client: Client, args: dict) -> CommandResults:
+    inventory_id = args.pop('inventory_id', None)
+    url_suffix = f'inventories/{inventory_id}/hosts/'
+
+    body = {'name': args.get('host_name'),
+            'description': args.get('description', ''),
+            'enabled': bool(args.get('enabled', 'True')),
+            }
+    response = client.api_request(method='POST', url_suffix=url_suffix, json_data=body)
+    response.pop('related', None)  # remove irrelevant fields from output
+    response.pop('summary_fields', None)
```

Comment:
as you remove it in the majority of the calls, maybe better to do that in the client and handle it if it is not in the `response`.